tests/mcb/test_mcb.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from ingenialink.ethernet.register import REG_DTYPE
+from ingenialink.utils._utils import convert_dtype_to_bytes, \
+    convert_bytes_to_dtype
+from ingenialink.utils.mcb import MCB
+from ingenialink.constants import MCB_CMD_READ, MCB_CMD_WRITE
+from ingenialink.exceptions import ILWrongCRCError, ILNACKError,\
+    ILWrongRegisterError
+
+
+@pytest.mark.parametrize("cmd, subnode, address, data, reg_dtype, extended, result", [
+    (MCB_CMD_READ, 1, 0x630, None, REG_DTYPE.FLOAT, False, 'a100026300000000000000009fcc'),
+    (MCB_CMD_WRITE, 1, 0x630, 25.5, REG_DTYPE.FLOAT, False, 'a10004630000cc4100000000cab1'),
+    (MCB_CMD_WRITE, 1, 0x6E5, "http://www.ingeniamc.com", REG_DTYPE.STR, True,
+     'a100556e1800000000000000b44b687474703a2f2f7777772e696e67656e69616d632e636f6d')
+])
+def test_build_mcb_frame(cmd, subnode, address, data, reg_dtype, extended, result):
+    default_frame_len = MCB.MCB_FRAME_SIZE
+    if data is not None:
+        data = convert_dtype_to_bytes(data, reg_dtype)
+    frame = MCB.build_mcb_frame(cmd, subnode, address, data)
+    if extended:
+        assert len(frame) > default_frame_len
+    else:
+        assert len(frame) == default_frame_len
+    assert result == frame.hex()
+
+
+@pytest.mark.parametrize("expected_address, data, reg_dtype, frame", [
+    (0x6e5, "http://www.ingeniamc.com", REG_DTYPE.STR, 'a100576e18000000000000003e9568'
+                                                       '7474703a2f2f7777772e696e67656e6'
+                                                       '9616d632e636f6d')
+])
+def test_read_mcb_frame(expected_address, data, reg_dtype, frame):
+    frame_byte_arr = bytearray.fromhex(frame)
+    data_bytes = MCB.read_mcb_data(expected_address, frame_byte_arr)
+    assert data == convert_bytes_to_dtype(data_bytes, reg_dtype)
+
+
+@pytest.mark.parametrize("expected_address, frame", [
```

Comment:
The same at [this comment](https://github.com/ingeniamc/ingenialink-python/pull/221/files#r998444141)